swift/Glacier2/callback/Sources/Callback.ice

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#pragma once
+
+module Demo
+{
+interface CallbackReceiver
+{
+void callback();
+}
+
+interface Callback
+{
+void initiateCallback(CallbackReceiver* proxy);
+void shutdown();
+}
+}
```

Comment:
This file is not correctly indented , see https://github.com/zeroc-ice/ice-demos/blob/3.7/csharp/Glacier2/callback/Callback.ice